linux_os/guide/system/software/gnome/gnome_login_screen/gnome_gdm_disable_unattended_automatic_login/oval/shared.xml

```diff
@@ -0,0 +1,35 @@
+<def-group>
+  <definition class="compliance" id="gnome_gdm_disable_unattended_automatic_login" version="2">
+    {{{ oval_metadata("Disable the GNOME Display Manager (GDM) ability to allow users to
+      automatically login.") }}}
+    <criteria operator="AND">
+      <criterion comment="Disable GDM Automatic Login" test_ref="test_disable_automatic_login" />
+      <criterion comment="Disable GDM Automatic Login" test_ref="test_disable_unattended_login" />
```

Comment:
They both have the same comment which makes them more difficult to distinguish.
```suggestion
      <criterion comment="Disable GDM Password Less Login" test_ref="test_disable_unattended_login" />
```

Reply:
Fixed